### PR TITLE
review comments on NHWC doc

### PR DESCRIPTION
**Pull request by jjsjann123**

### CODE REVIEW

**Comment by jjsjann123** [comment]
Review comments put together by the PyTorch team at NV.
cc'ing people in the loop @ngimel @VitalyFedyunin @mcarilli @ptrblck

File: intermediate_source/memory_format_tutorial.py [hunk 1]
```diff
@@ -84,29 +118,51 @@
 print(y.stride()) # Ouputs: (3072, 1, 96, 3)
 
 ######################################################################
-# Pointwise operators preserves memory format
+# Pointwise operators preserves memory format. Note: for p-wise
+# operatros with multiple inputs of different memory format,
+# contiguous memory format is the dominating memory format. This means
```
Comment (VitalyFedyunin) [inline intermediate_source/memory_format_tutorial.py:123]: Going to be changed in https://github.com/pytorch/pytorch/pull/37968

**Comment by VitalyFedyunin** [comment]
@jjsjann123 looks awesome, with just one small comment.

File: intermediate_source/memory_format_tutorial.py (hunk 1 above)
Comment (jjsjann123) [reply intermediate_source/memory_format_tutorial.py:123]: Thanks. The code change does spit out a warning of output format.

I'll remove this section then.

**Comment by jjsjann123** [comment]
Ping @VitalyFedyunin after merge master.

**Comment by jjsjann123** [comment]
Bump again. @VitalyFedyunin do we still have concern on updating the doc?

**Pull request merged**